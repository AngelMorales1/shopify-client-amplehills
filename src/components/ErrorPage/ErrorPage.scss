@import 'settings';

.ErrorPage {
  &__logo {
    max-width: spacing(40);
  }
}
