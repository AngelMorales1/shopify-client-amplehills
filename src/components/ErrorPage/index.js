import React from 'react';
import cx from 'classnames';

import styles from './ErrorPage.scss';
import { Image, Button } from 'components/base';

const ErrorPage = ({ children }) => {
  console.log('Error page rendered');

  return (
    <div className={cx(styles['ErrorPage'], 'fixed-cover bg-peach')}>
      {children ? (
        children
      ) : (
        <div className="absolute-cover flex items-center justify-center">
          <div className="center text-white transition-slide-up-large">
            <Image
              className={cx(styles['ErrorPage__logo'], 'w100')}
              src="/assets/images/ample-hills-logo.svg"
            />
            <div className="my3 text-container-width p2">
              <h1 className="title carter mb2">{"We'll be right back!"}</h1>
              <p className="block-subheadline">
                Our online shop is currently unavailable. In the meantime, feel
                free to reach out at{' '}
                <a
                  href="mailto:info@amplehills.com"
                  target="_blank"
                  rel="noopener noreferrer"
                >
                  info@amplehills.com
                </a>{' '}
                or find one of{' '}
                <a
                  href="https://www.google.com/search?q=ample+hills+near+me"
                  target="_blank"
                  rel="noopener noreferrer"
                >
                  our scoop shops
                </a>.
              </p>
            </div>
            <Button
              className="mx-auto"
              color="white-peach"
              variant="primary"
              label="Refresh Page"
              onClick={() => window.location.reload()}
            />
          </div>
          <div className="absolute b0 l0 w100">
            <Image src="/assets/images/ample-hills-walt-walk-animation.gif" />
          </div>
        </div>
      )}
    </div>
  );
};

export default ErrorPage;
