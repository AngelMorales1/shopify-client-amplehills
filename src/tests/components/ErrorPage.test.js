import React from 'react';
import { shallow } from 'enzyme';

import ErrorPage from 'components/ErrorPage';

it('renders without data', () => {
  const component = shallow(<ErrorPage />);

  expect(component).toMatchSnapshot();
});

it('renders with data', () => {
  const component = shallow(
    <ErrorPage>
      <p>foo</p>
    </ErrorPage>
  );

  expect(component).toMatchSnapshot();
});
