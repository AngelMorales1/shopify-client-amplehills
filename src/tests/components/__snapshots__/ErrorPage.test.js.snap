// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ErrorPage>
    <p>
        foo
    </p>
</ErrorPage>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        foo
</p>,
      "className": "fixed-cover bg-peach",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "foo",
      },
      "ref": null,
      "rendered": "foo",
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          foo
</p>,
        "className": "fixed-cover bg-peach",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "foo",
        },
        "ref": null,
        "rendered": "foo",
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`renders without data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ErrorPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="absolute-cover flex items-center justify-center"
>
        <div
                className="center text-white transition-slide-up-large"
        >
                <Image
                        alt=""
                        bg={false}
                        className="w100"
                        onImgLoad={[Function]}
                        src="/assets/images/ample-hills-logo.svg"
                        style={Object {}}
                />
                <div
                        className="my3 text-container-width p2"
                >
                        <h1
                                className="title carter mb2"
                        >
                                We'll be right back!
                        </h1>
                        <p
                                className="block-subheadline"
                        >
                                Our online shop is currently unavailable. In the meantime, feel free to reach out at
                                 
                                <a
                                        href="mailto:info@amplehills.com"
                                        rel="noopener noreferrer"
                                        target="_blank"
                                >
                                        info@amplehills.com
                                </a>
                                 
                                or find one of
                                 
                                <a
                                        href="https://www.google.com/search?q=ample+hills+near+me"
                                        rel="noopener noreferrer"
                                        target="_blank"
                                >
                                        our scoop shops
                                </a>
                                .
                        </p>
                </div>
                <Button
                        className="mx-auto"
                        color="white-peach"
                        disabled={false}
                        fullWidth={false}
                        hover="shadow"
                        label="Refresh Page"
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onMouseLeave={[Function]}
                        shadow={false}
                        to=""
                        type="button"
                        variant="primary"
                />
        </div>
        <div
                className="absolute b0 l0 w100"
        >
                <Image
                        alt=""
                        bg={false}
                        className="w100"
                        onImgLoad={[Function]}
                        src="/assets/images/ample-hills-walt-walk-animation.gif"
                        style={Object {}}
                />
        </div>
</div>,
      "className": "fixed-cover bg-peach",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="center text-white transition-slide-up-large"
>
            <Image
                        alt=""
                        bg={false}
                        className="w100"
                        onImgLoad={[Function]}
                        src="/assets/images/ample-hills-logo.svg"
                        style={Object {}}
            />
            <div
                        className="my3 text-container-width p2"
            >
                        <h1
                                    className="title carter mb2"
                        >
                                    We'll be right back!
                        </h1>
                        <p
                                    className="block-subheadline"
                        >
                                    Our online shop is currently unavailable. In the meantime, feel free to reach out at
                                     
                                    <a
                                                href="mailto:info@amplehills.com"
                                                rel="noopener noreferrer"
                                                target="_blank"
                                    >
                                                info@amplehills.com
                                    </a>
                                     
                                    or find one of
                                     
                                    <a
                                                href="https://www.google.com/search?q=ample+hills+near+me"
                                                rel="noopener noreferrer"
                                                target="_blank"
                                    >
                                                our scoop shops
                                    </a>
                                    .
                        </p>
            </div>
            <Button
                        className="mx-auto"
                        color="white-peach"
                        disabled={false}
                        fullWidth={false}
                        hover="shadow"
                        label="Refresh Page"
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onMouseLeave={[Function]}
                        shadow={false}
                        to=""
                        type="button"
                        variant="primary"
            />
</div>,
          <div
            className="absolute b0 l0 w100"
>
            <Image
                        alt=""
                        bg={false}
                        className="w100"
                        onImgLoad={[Function]}
                        src="/assets/images/ample-hills-walt-walk-animation.gif"
                        style={Object {}}
            />
</div>,
        ],
        "className": "absolute-cover flex items-center justify-center",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <Image
                alt=""
                bg={false}
                className="w100"
                onImgLoad={[Function]}
                src="/assets/images/ample-hills-logo.svg"
                style={Object {}}
/>,
              <div
                className="my3 text-container-width p2"
>
                <h1
                                className="title carter mb2"
                >
                                We'll be right back!
                </h1>
                <p
                                className="block-subheadline"
                >
                                Our online shop is currently unavailable. In the meantime, feel free to reach out at
                                 
                                <a
                                                href="mailto:info@amplehills.com"
                                                rel="noopener noreferrer"
                                                target="_blank"
                                >
                                                info@amplehills.com
                                </a>
                                 
                                or find one of
                                 
                                <a
                                                href="https://www.google.com/search?q=ample+hills+near+me"
                                                rel="noopener noreferrer"
                                                target="_blank"
                                >
                                                our scoop shops
                                </a>
                                .
                </p>
</div>,
              <Button
                className="mx-auto"
                color="white-peach"
                disabled={false}
                fullWidth={false}
                hover="shadow"
                label="Refresh Page"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                shadow={false}
                to=""
                type="button"
                variant="primary"
/>,
            ],
            "className": "center text-white transition-slide-up-large",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "alt": "",
                "bg": false,
                "children": null,
                "className": "w100",
                "onImgLoad": [Function],
                "src": "/assets/images/ample-hills-logo.svg",
                "style": Object {},
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h1
                    className="title carter mb2"
>
                    We'll be right back!
</h1>,
                  <p
                    className="block-subheadline"
>
                    Our online shop is currently unavailable. In the meantime, feel free to reach out at
                     
                    <a
                                        href="mailto:info@amplehills.com"
                                        rel="noopener noreferrer"
                                        target="_blank"
                    >
                                        info@amplehills.com
                    </a>
                     
                    or find one of
                     
                    <a
                                        href="https://www.google.com/search?q=ample+hills+near+me"
                                        rel="noopener noreferrer"
                                        target="_blank"
                    >
                                        our scoop shops
                    </a>
                    .
</p>,
                ],
                "className": "my3 text-container-width p2",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "We'll be right back!",
                    "className": "title carter mb2",
                  },
                  "ref": null,
                  "rendered": "We'll be right back!",
                  "type": "h1",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      "Our online shop is currently unavailable. In the meantime, feel free to reach out at",
                      " ",
                      <a
                        href="mailto:info@amplehills.com"
                        rel="noopener noreferrer"
                        target="_blank"
>
                        info@amplehills.com
</a>,
                      " ",
                      "or find one of",
                      " ",
                      <a
                        href="https://www.google.com/search?q=ample+hills+near+me"
                        rel="noopener noreferrer"
                        target="_blank"
>
                        our scoop shops
</a>,
                      ".",
                    ],
                    "className": "block-subheadline",
                  },
                  "ref": null,
                  "rendered": Array [
                    "Our online shop is currently unavailable. In the meantime, feel free to reach out at",
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "info@amplehills.com",
                        "href": "mailto:info@amplehills.com",
                        "rel": "noopener noreferrer",
                        "target": "_blank",
                      },
                      "ref": null,
                      "rendered": "info@amplehills.com",
                      "type": "a",
                    },
                    " ",
                    "or find one of",
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "our scoop shops",
                        "href": "https://www.google.com/search?q=ample+hills+near+me",
                        "rel": "noopener noreferrer",
                        "target": "_blank",
                      },
                      "ref": null,
                      "rendered": "our scoop shops",
                      "type": "a",
                    },
                    ".",
                  ],
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": null,
                "className": "mx-auto",
                "color": "white-peach",
                "disabled": false,
                "fullWidth": false,
                "hover": "shadow",
                "label": "Refresh Page",
                "onClick": [Function],
                "onMouseEnter": [Function],
                "onMouseLeave": [Function],
                "shadow": false,
                "to": "",
                "type": "button",
                "variant": "primary",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Image
              alt=""
              bg={false}
              className="w100"
              onImgLoad={[Function]}
              src="/assets/images/ample-hills-walt-walk-animation.gif"
              style={Object {}}
/>,
            "className": "absolute b0 l0 w100",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "alt": "",
              "bg": false,
              "children": null,
              "className": "w100",
              "onImgLoad": [Function],
              "src": "/assets/images/ample-hills-walt-walk-animation.gif",
              "style": Object {},
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="absolute-cover flex items-center justify-center"
>
          <div
                    className="center text-white transition-slide-up-large"
          >
                    <Image
                              alt=""
                              bg={false}
                              className="w100"
                              onImgLoad={[Function]}
                              src="/assets/images/ample-hills-logo.svg"
                              style={Object {}}
                    />
                    <div
                              className="my3 text-container-width p2"
                    >
                              <h1
                                        className="title carter mb2"
                              >
                                        We'll be right back!
                              </h1>
                              <p
                                        className="block-subheadline"
                              >
                                        Our online shop is currently unavailable. In the meantime, feel free to reach out at
                                         
                                        <a
                                                  href="mailto:info@amplehills.com"
                                                  rel="noopener noreferrer"
                                                  target="_blank"
                                        >
                                                  info@amplehills.com
                                        </a>
                                         
                                        or find one of
                                         
                                        <a
                                                  href="https://www.google.com/search?q=ample+hills+near+me"
                                                  rel="noopener noreferrer"
                                                  target="_blank"
                                        >
                                                  our scoop shops
                                        </a>
                                        .
                              </p>
                    </div>
                    <Button
                              className="mx-auto"
                              color="white-peach"
                              disabled={false}
                              fullWidth={false}
                              hover="shadow"
                              label="Refresh Page"
                              onClick={[Function]}
                              onMouseEnter={[Function]}
                              onMouseLeave={[Function]}
                              shadow={false}
                              to=""
                              type="button"
                              variant="primary"
                    />
          </div>
          <div
                    className="absolute b0 l0 w100"
          >
                    <Image
                              alt=""
                              bg={false}
                              className="w100"
                              onImgLoad={[Function]}
                              src="/assets/images/ample-hills-walt-walk-animation.gif"
                              style={Object {}}
                    />
          </div>
</div>,
        "className": "fixed-cover bg-peach",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="center text-white transition-slide-up-large"
>
              <Image
                            alt=""
                            bg={false}
                            className="w100"
                            onImgLoad={[Function]}
                            src="/assets/images/ample-hills-logo.svg"
                            style={Object {}}
              />
              <div
                            className="my3 text-container-width p2"
              >
                            <h1
                                          className="title carter mb2"
                            >
                                          We'll be right back!
                            </h1>
                            <p
                                          className="block-subheadline"
                            >
                                          Our online shop is currently unavailable. In the meantime, feel free to reach out at
                                           
                                          <a
                                                        href="mailto:info@amplehills.com"
                                                        rel="noopener noreferrer"
                                                        target="_blank"
                                          >
                                                        info@amplehills.com
                                          </a>
                                           
                                          or find one of
                                           
                                          <a
                                                        href="https://www.google.com/search?q=ample+hills+near+me"
                                                        rel="noopener noreferrer"
                                                        target="_blank"
                                          >
                                                        our scoop shops
                                          </a>
                                          .
                            </p>
              </div>
              <Button
                            className="mx-auto"
                            color="white-peach"
                            disabled={false}
                            fullWidth={false}
                            hover="shadow"
                            label="Refresh Page"
                            onClick={[Function]}
                            onMouseEnter={[Function]}
                            onMouseLeave={[Function]}
                            shadow={false}
                            to=""
                            type="button"
                            variant="primary"
              />
</div>,
            <div
              className="absolute b0 l0 w100"
>
              <Image
                            alt=""
                            bg={false}
                            className="w100"
                            onImgLoad={[Function]}
                            src="/assets/images/ample-hills-walt-walk-animation.gif"
                            style={Object {}}
              />
</div>,
          ],
          "className": "absolute-cover flex items-center justify-center",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Image
                  alt=""
                  bg={false}
                  className="w100"
                  onImgLoad={[Function]}
                  src="/assets/images/ample-hills-logo.svg"
                  style={Object {}}
/>,
                <div
                  className="my3 text-container-width p2"
>
                  <h1
                                    className="title carter mb2"
                  >
                                    We'll be right back!
                  </h1>
                  <p
                                    className="block-subheadline"
                  >
                                    Our online shop is currently unavailable. In the meantime, feel free to reach out at
                                     
                                    <a
                                                      href="mailto:info@amplehills.com"
                                                      rel="noopener noreferrer"
                                                      target="_blank"
                                    >
                                                      info@amplehills.com
                                    </a>
                                     
                                    or find one of
                                     
                                    <a
                                                      href="https://www.google.com/search?q=ample+hills+near+me"
                                                      rel="noopener noreferrer"
                                                      target="_blank"
                                    >
                                                      our scoop shops
                                    </a>
                                    .
                  </p>
</div>,
                <Button
                  className="mx-auto"
                  color="white-peach"
                  disabled={false}
                  fullWidth={false}
                  hover="shadow"
                  label="Refresh Page"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  shadow={false}
                  to=""
                  type="button"
                  variant="primary"
/>,
              ],
              "className": "center text-white transition-slide-up-large",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "alt": "",
                  "bg": false,
                  "children": null,
                  "className": "w100",
                  "onImgLoad": [Function],
                  "src": "/assets/images/ample-hills-logo.svg",
                  "style": Object {},
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h1
                      className="title carter mb2"
>
                      We'll be right back!
</h1>,
                    <p
                      className="block-subheadline"
>
                      Our online shop is currently unavailable. In the meantime, feel free to reach out at
                       
                      <a
                                            href="mailto:info@amplehills.com"
                                            rel="noopener noreferrer"
                                            target="_blank"
                      >
                                            info@amplehills.com
                      </a>
                       
                      or find one of
                       
                      <a
                                            href="https://www.google.com/search?q=ample+hills+near+me"
                                            rel="noopener noreferrer"
                                            target="_blank"
                      >
                                            our scoop shops
                      </a>
                      .
</p>,
                  ],
                  "className": "my3 text-container-width p2",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "We'll be right back!",
                      "className": "title carter mb2",
                    },
                    "ref": null,
                    "rendered": "We'll be right back!",
                    "type": "h1",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        "Our online shop is currently unavailable. In the meantime, feel free to reach out at",
                        " ",
                        <a
                          href="mailto:info@amplehills.com"
                          rel="noopener noreferrer"
                          target="_blank"
>
                          info@amplehills.com
</a>,
                        " ",
                        "or find one of",
                        " ",
                        <a
                          href="https://www.google.com/search?q=ample+hills+near+me"
                          rel="noopener noreferrer"
                          target="_blank"
>
                          our scoop shops
</a>,
                        ".",
                      ],
                      "className": "block-subheadline",
                    },
                    "ref": null,
                    "rendered": Array [
                      "Our online shop is currently unavailable. In the meantime, feel free to reach out at",
                      " ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "info@amplehills.com",
                          "href": "mailto:info@amplehills.com",
                          "rel": "noopener noreferrer",
                          "target": "_blank",
                        },
                        "ref": null,
                        "rendered": "info@amplehills.com",
                        "type": "a",
                      },
                      " ",
                      "or find one of",
                      " ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "our scoop shops",
                          "href": "https://www.google.com/search?q=ample+hills+near+me",
                          "rel": "noopener noreferrer",
                          "target": "_blank",
                        },
                        "ref": null,
                        "rendered": "our scoop shops",
                        "type": "a",
                      },
                      ".",
                    ],
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": null,
                  "className": "mx-auto",
                  "color": "white-peach",
                  "disabled": false,
                  "fullWidth": false,
                  "hover": "shadow",
                  "label": "Refresh Page",
                  "onClick": [Function],
                  "onMouseEnter": [Function],
                  "onMouseLeave": [Function],
                  "shadow": false,
                  "to": "",
                  "type": "button",
                  "variant": "primary",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Image
                alt=""
                bg={false}
                className="w100"
                onImgLoad={[Function]}
                src="/assets/images/ample-hills-walt-walk-animation.gif"
                style={Object {}}
/>,
              "className": "absolute b0 l0 w100",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "alt": "",
                "bg": false,
                "children": null,
                "className": "w100",
                "onImgLoad": [Function],
                "src": "/assets/images/ample-hills-walt-walk-animation.gif",
                "style": Object {},
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
